src/plugin.js

```diff
@@ -163,7 +163,9 @@ const plugin = {
   },
   defaultOptions: {
     printWidth: 80,
-    tabWidth: 2
+    tabWidth: 2,
+    trailingComma: "es5",
```

Comment:
```suggestion
    trailingComma: "none",
```

Reply:
but prettier does by default add trailing commas, and it feels like we should be matching the defaults of prettier?

Reply:
We're not trying to match prettier style, we're trying to be idiomatic for Ruby. That's why the defaultoptions is there for plugins. For example, the PHP plugin uses 4 spaces by default for tabs (https://github.com/prettier/plugin-php/blob/3061289e50228149359bf8284be318ee042eadfa/src/index.js#L152). 

Besides, switching to this would mean a breaking change and another major revision.

Reply:
great point, thanks :) 

i've made that change now